docs/pages/2019_MARVEL_Psik_MaX/sections/bands.rst

```diff
@@ -14,32 +14,42 @@ calculation steps, are performed automatically by the WorkChain.
 
 The demonstration of the workchain will be performed in a Jupyter
 notebook. To run it, follow the instructions that were given for the
-querybuilder notebook in section [sec:querybuilder]. The only difference
+querybuilder notebook in section **MISSINGQUERYBUILDERREF**. The only difference
```

Comment:
you're still planning to add this reference, right?
if a label in the querybuilder section is missing you can just add it

Reply:
Yes! I just did not want to touch work in progress from other people, but remember to add it. Sorry about the non-standard self-reminder